frameworks/OCaml/webmachine/haproxy.cfg

```diff
@@ -7,10 +7,10 @@ global
     nbthread    2
 
 defaults
-    mode                    http
+    mode                    tcp
```

Comment:
We should probably remove haproxy from the OCaml benchmarks, now that we know that sharing a socket yields better throughput

Reply:
In the long term yes but I would like to have one run completed to see how much overhead it adds. Locally the plaintext test actually produces a higher RPS behind haproxy than with the multiprocess approach.